pkg/updater/inflate.go

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package updater
+
+import (
+	"context"
+	"time"
+
+	"github.com/GoogleCloudPlatform/testgrid/pb/state"
+)
+
+// inflatedColumn holds all the entries for a given column.
+//
+// This includes both:
+// * Column state metadata and
+// * cell values for every row in this column
+type inflatedColumn struct {
+	column *state.Column
+	cells  map[string]cell
+}
+
+// cell holds row values for a given column
+type cell struct {
```

Comment:
nit: `cell` sounds confusing since I typically think of a single square for that. (If Go's thing weren't called a `slice` I would suggest `slice` or `columnSlice` or similar).

Reply:
Updated the doc, this is just a square (the inflatedColumns has a map[string]cell)